caiman/source_extraction/volpy/mrcnn/model.py

```diff
@@ -17,6 +17,10 @@
 import multiprocessing
 import numpy as np
 import tensorflow as tf
+
+config = tf.ConfigProto()
```

Comment:
we'll need to be very careful not to import volpy from anywhere else in the source tree, because this change has the import initialise GPU devices, rather than calling functions.

Consider not doing this outside of the functions in the volpy codebase, to prevent that "executes code on import" behaviour.

Reply:
I delete these lines. Not important.